net-mail/gnubiff/files/gnubiff-2.2.17-clang.patch

```diff
@@ -0,0 +1,15 @@
+--- gnubiff-2.2.17/src/Makefile.am.orig	2016-06-12 04:18:12.000000000 -0400
```

Comment:
Please apply `scrub-patch` (https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto)
and add short description mentioning #930500